proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -120,4 +122,5 @@ message MappingMetadata {
   string map_name = 11; // name of the map package
   string slam_version = 12; // version tag from request, defaults to stable
   string config = 13; // a robot config for a slam session
+  string end_status_msg = 14; // additional details on the end status if needed, such as errors
```

Comment:
we did not include this in the scope doc, but should we have `end_status_msg` in the metadata too?

Reply:
Yes, that was the intent from @sbal13 's PR: https://github.com/viamrobotics/app/pull/2213#issuecomment-1681188209!

I guess we missed that in the scope doc update